sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveClientImpl.scala

```diff
@@ -186,7 +187,12 @@ private[hive] class HiveClientImpl(
   /** Returns the configuration for the current session. */
   def conf: HiveConf = state.getConf
 
-  private val userName = conf.getUser
```

Comment:
this source code of conf.getUser is:
`public String getUser() throws IOException {`
`        try {`
`             UserGroupInformation ugi = Utils.getUGI();` 
`             return ugi.getUserName();` 
`         } catch (LoginException var2) {` 
`             throw new IOException(var2);` 
`         }` 
`     }` 
this chang just use ugi.getShortUserName instead of ugi.getUserName()